src/core/blob.cpp

```diff
@@ -20,31 +26,37 @@ blob::blob(session & s)
 
 blob::~blob()
 {
-    delete backEnd_;
+    if (backEnd_)
```

Comment:
This change is harmless but not necessary, deleting a null pointer is a perfectly valid operation (doing nothing).